app/models/katello/purpose_status.rb

```diff
@@ -38,6 +38,7 @@ def self.to_label(status)
     end
 
     def self.to_status(status, purpose_method, options)
+      return nil if status.host.organization.simple_content_access?
```

Comment:
What does this do?

Reply:
I don't even remember why 😆, confirmed with my delete it still works correctly, so just removing it.